exporters/stdout/stdoutlog/exporter_test.go

```diff
@@ -0,0 +1,227 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package stdoutlog_test // import "go.opentelemetry.io/otel/exporters/stdout/stdoutout"
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.opentelemetry.io/otel/exporters/stdout/stdoutlog"
+	"go.opentelemetry.io/otel/log"
+	sdklog "go.opentelemetry.io/otel/sdk/log"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func TestExporter(t *testing.T) {
+	var buf bytes.Buffer
+
+	exporter, err := stdoutlog.New(stdoutlog.WithWriter(&buf))
+	assert.NoError(t, err)
+	assert.NotNil(t, exporter)
+
+	now := time.Now()
+	record := getRecord(now)
+
+	// Export a record
+	err = exporter.Export(context.Background(), []sdklog.Record{record})
```

Comment:
The exported record output is not checked.

Reply:
It actually checked in https://github.com/open-telemetry/opentelemetry-go/blob/50164f41a7a8ccd2545b1b4894d17f949de2bc6f/exporters/stdout/stdoutlog/exporter_test.go#L48

But it is not that clear. I have added another check right below the export.